bot/exts/evergreen/space.py

```diff
@@ -74,11 +74,18 @@ def cog_unload(self) -> None:
         # Parse date to params, when provided. Show error message when invalid formatting
         if date:
             try:
-                params["date"] = datetime.strptime(date, "%Y-%m-%d").date().isoformat()
+                apod_date = datetime.strptime(date, "%Y-%m-%d").date()
             except ValueError:
                 await ctx.send(f"Invalid date {date}. Please make sure your date is in format YYYY-MM-DD.")
                 return
 
+            now = datetime.now().date()
+            if datetime(1995, 6, 16).date() > apod_date or now < apod_date:
+                await ctx.send(f"Date must be in range 16th June 1995 and {now.strftime('%dth %B %Y')}.")
```

Comment:
* If I'm not mistaken, this instructs the user to provide the date in an unsupported format.
* The current format string produces `07th April 2020` today.
* I think `Date must be between 16th June 1995 and today` would be sufficient.